4-WebApp-your-API/4-1-MyOrg/Client/Services/TodoListService.cs

```diff
@@ -58,7 +58,7 @@ public async Task<Todo> AddAsync(Todo todo)
                 return todo;
             }
 
-            throw new HttpRequestException($"Invalid status code in the HttpResponseMessage: {response.StatusCode}.");
```

Comment:
>throw new HttpRequestException($"Invalid status code in the HttpResponseMessage: {response.StatusCode}."); [](http://example.com/codeflow?start=12&length=106)

why?????????????? why hide the error ???

Reply:
I think you're looking into old code